tests/_utils-tests/model.js

```diff
@@ -460,61 +453,61 @@ describe( 'model test utils', () => {
 		it( 'throws when unexpected closing tag', () => {
 			expect( () => {
 				parse( '<a><b></a></b>' );
-			} ).to.throw( Error, 'Parse error - unexpected closing tag.' );
+			} ).to.throw( Error );
 		} );
 
 		it( 'throws when unexpected attribute', () => {
 			expect( () => {
 				parse( '<a ?></a>' );
-			} ).to.throw( Error, 'Parse error - unexpected token: ?.' );
+			} ).to.throw( Error );
 		} );
 
 		it( 'throws when incorrect tag', () => {
 			expect( () => {
 				parse( '<a' );
-			} ).to.throw( Error, 'Parse error - unexpected token: <a.' );
+			} ).to.throw( Error );
 		} );
 
 		it( 'throws when missing closing tag', () => {
 			expect( () => {
 				parse( '<a><b></b>' );
-			} ).to.throw( Error, 'Parse error - missing closing tags: a.' );
+			} ).to.throw( Error );
 		} );
 
 		it( 'throws when missing opening tag for text', () => {
 			expect( () => {
 				parse( '</$text>' );
-			} ).to.throw( Error, 'Parse error - unexpected closing tag.' );
+			} ).to.throw( Error );
 		} );
 
 		it( 'throws when missing closing tag for text', () => {
 			expect( () => {
 				parse( '<$text>' );
-			} ).to.throw( Error, 'Parse error - missing closing tags: $text.' );
+			} ).to.throw( Error );
 		} );
```

Comment:
Do we need all these tests now? Aren't they check native behavior now? Maybe one will be enough then.